packages/grid/data-grid/src/tests/pagination.DataGrid.test.tsx

```diff
@@ -187,5 +193,50 @@ describe('<DataGrid /> - Pagination', () => {
       fireEvent.click(screen.getByRole('button', { name: /next page/i }));
       expect(getColumnValues()).to.deep.equal(['Nike 1']);
     });
+
+    describe('AutoPageSize', () => {
```

Comment:
A naming convention we have used in the main repo, I guess it makes sense to follow it here too
```suggestion
    describe('prop: autoPageSize', () => {
```